app/src/main/java/org/wikipedia/commons/ImageTagsProvider.kt

```diff
@@ -0,0 +1,38 @@
+package org.wikipedia.commons
+
+import io.reactivex.Observable
+import io.reactivex.schedulers.Schedulers
+import org.wikipedia.dataclient.Service
+import org.wikipedia.dataclient.ServiceFactory
+import org.wikipedia.dataclient.WikiSite
+import java.util.*
+
+object ImageTagsProvider {
+    fun getImageTagsObservable(pageId: String, langCode: String): Observable<Map<String, List<String>>> {
```

Comment:
`pageId` should be a `Long`.

Reply:
I check the `pageId` in `MwQueryPage`, it is a `int` type. Can we use that?